src/UI/App.hs

```diff
@@ -49,11 +49,14 @@ import UI.Views
        (indexView, mailView, composeView, helpView, listOfMailsView,
         filebrowserView, focusedViewWidget, focusedViewWidgets,
         focusedViewName)
-import UI.ComposeEditor.Main (attachmentsEditor, drawHeaders)
+import UI.ComposeEditor.Main (attachmentsEditor, drawHeaders, renderConfirm)
 import Types
 
 drawUI :: AppState -> [Widget Name]
-drawUI s = [vBox (renderWidget s (focusedViewName s) <$> focusedViewWidgets s)]
+drawUI s = let ui = vBox (renderWidget s (focusedViewName s) <$> focusedViewWidgets s)
```

Comment:
Currently we're rendering all UI sequentially TV style, top to bottom. This works 99% of the time, except when we want to render a widget on top of everything. I don't know how often this will happen or if the dialog is the only instance. So this is obviously a hack at the moment :/